src/Mobs/Wolf.cpp

```diff
@@ -337,6 +372,8 @@ void cWolf::Tick(std::chrono::milliseconds a_Dt, cChunk & a_Chunk)
 	{
 		StopMovingToPosition();
 	}
+
+	cMonster::LoveTick();
```

Comment:
This needs a bit of testing - this way loving takes priority over all other behaviours, is that correct in vanilla? It could be that LoveTick needs to be moved more to the start of the function.

Reply:
I did some testing...

The behaviour is a bit strage:

The wolfes run into each other and then attack the enemy and then run into each other a create an offspring.